projects/wvr-elements/src/lib/shared/animation/wvr-animation.service.ts

```diff
@@ -0,0 +1,108 @@
+import { Injectable, ɵConsole } from '@angular/core';
```

Comment:
This is a duplicate file and should be deleted. Root singleton services below in the core directory.